app/src/main/res/values/strings.xml

```diff
@@ -574,4 +574,7 @@
   <string name="administrator_controls_fragment_test_activity_label">Administrator Controls Fragment Test Activity</string>
   <string name="select_a_topic_to_start">Select a Topic to Start</string>
   <string name="revision_navigation_cards_header">Continue Studying</string>
+  <string name="please_select_all_correct_choices">Please select all correct choices</string>
+  <string name="you_may_select_more_choices">You may select more choices</string>
+  <string name="no_more_than_choices_may_be_selected">No more than %s choices may be selected.</string>
```

Comment:
Are these exact copies from Oppia web? The last one has punctuation but the first two don't which seems like an inconsistency.